jaeger-core/src/main/java/io/jaegertracing/internal/JaegerSpanContext.java

```diff
@@ -68,15 +68,15 @@ public String getBaggageItem(String key) {
     return this.baggage;
   }
 
-  public long getTraceId() {
+  public TraceId getTraceId() {
```

Comment:
This probably breaks backwards-compatibility. Is there a way to preserve the return type? I realize for `TraceId` this may be difficult, but why change `long` to `SpanId` unnecessarily?

Reply:
I believe it makes the API cleaner to have SpanId and TraceId as returned type.
I think keeping long for span and TraceId for traces is a short term solution that might change if spanIds become 128 bits for any reason. I would rather encapsulate now than later. What do you think ?

We can also change the method name and have both.